R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -885,24 +894,24 @@ test_that("collect() and take() on a DataFrame return the same number of rows an
 })
 
 test_that("collect() support Unicode characters", {
-  lines <- c("{\"name\":\"안녕하세요\"}",
-             "{\"name\":\"您好\", \"age\":30}",
-             "{\"name\":\"こんにちは\", \"age\":19}",
-             "{\"name\":\"Xin chào\"}")
+  lines <- c("{\"name\":\"?????\"}",
+             "{\"name\":\"??\", \"age\":30}",
+             "{\"name\":\"?????\", \"age\":19}",
+             "{\"name\":\"Xin ch?o\"}")
```

Comment:
Is it okay to replace these unicode characters?

Reply:
no, we should not change these